x-pack/plugins/ml/public/application/routing/routes/settings/calendar_new_edit.tsx

```diff
@@ -58,6 +57,9 @@ export const editCalendarRouteFactory = (
   basePath: string
 ): MlRoute => ({
   path: '/settings/calendars_list/edit_calendar/:calendarId',
+  title: i18n.translate('xpack.ml.settings.editCalendar.docTitle', {
```

Comment:
I'm getting 'Overview' as the page title when I edit a calendar:

![image](https://user-images.githubusercontent.com/7405507/151945302-7f5122d2-b1e1-4e7f-a789-ab6f2fc40ac8.png)

Same when I edit a filter list:

![image](https://user-images.githubusercontent.com/7405507/151945473-fb0ad06b-1aa6-43ba-8bd6-39e81e3b4e5e.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/124125/commits/8e54e359ad883ec4ca771ec6b856b98bda6be424